index.md

```diff
@@ -30,7 +30,18 @@ Nice, let's dive in. Click on the product below to get started.
         <a class="btn btn-guiding btn-outline btn-block mt-auto" href="/checkout/">{% icon shopping_cart %}&nbsp; Checkout</a>
       </div>
     </div>
-  </div>
+  </div> {% comment %}
```

Comment:
Why are we commenting out this section?

Reply:
@asbjornu wanted it to be not visible for everyone per now